tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -15,13 +15,14 @@
 
 from ducktape.mark.resource import cluster
 from ducktape.tests.test import Test
-from ducktape.mark import parametrize, ignore
+from ducktape.mark import matrix
 from kafkatest.services.kafka import KafkaService
 from kafkatest.services.zookeeper import ZookeeperService
 from kafkatest.services.streams import StreamsSmokeTestDriverService, StreamsSmokeTestJobRunnerService
-from kafkatest.version import LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11_0, DEV_BRANCH, KafkaVersion
+from kafkatest.version import LATEST_0_10_1, LATEST_0_10_2, LATEST_0_11, LATEST_1_0, DEV_BRANCH, KafkaVersion
 import time
 
+upgrade_versions = [str(LATEST_0_10_2), str(LATEST_0_11), str(LATEST_1_0), str(DEV_BRANCH)]
```

Comment:
Note: I had to drop 0.10.1 because `StreamsSmokeTest` had a different package name in that version (was: `org.apache.kafka.streams.smoketest.StreamsSmokeTest`.

I can put it back in, but it would require a conditional in the streams.py::StreamsSmokeTestBaseService. Or we can leave it out.

Thoughts?

Reply:
I would say leave it out, I believe the change from `0.10.1` to a higher version requires a full stop and can't be done from a rolling restart.

Reply:
Actually from `0.10.0` to `0.10.1` requires a full stop, from `0.10.1` to newer version does not. But I think it is still fine to leave it out.